docs/stylesheets/extra.css

```diff
@@ -117,59 +117,53 @@ the nav title is needed for backwards navigation in the collapsible
 nav variant.
 
 See https://github.com/astral-sh/uv/issues/5130 */
-.md-nav__title {
-  display: none;
-}
-@media screen and (max-width: 1219px) {
+@media screen and (min-width: 76.25em) {
```

Comment:
Where does this value come from?

Reply:
This value is from the main stylesheet, I figured it'd be best to match it though I think it is equivalent for most purposes?